test/cat.jl

```diff
@@ -137,15 +119,14 @@ module TestCat
     @test isequal(vcat(dtd, dtc0, dtc), dc)
     @test eltypes(vcat(dtd, dtc0)) == eltypes(dc)
 
-    # Missing columns
```

Comment:
I guess that was because of what this test precisely checks: missing columns. So whether to keep it or not depends on the behavior we choose.